static/app/components/sidebar/index.tsx

```diff
@@ -116,8 +116,11 @@ function Sidebar() {
   const collapsed = !!preferences.collapsed;
   const horizontal = useMedia(`(max-width: ${theme.breakpoints.medium})`);
 
-  // Avoid showing superuser UI on self-hosted instances
-  const hasSuperuserSession = isActiveSuperuser() && !ConfigStore.get('isSelfHosted');
+  // Avoid showing superuser UI on self-hosted instances or demo org
+  const hasSuperuserSession =
+    isActiveSuperuser() &&
+    !ConfigStore.get('isSelfHosted') &&
+    organization?.slug !== 'demo';
```

Comment:
```suggestion
```
passing in the organization into the hook should be enough. the hook in getsentry already covers the slug

Reply:
hooks are what we use to bring stuff in the getsentry repo into the sentry repo because they are separate (even though they get recombined to make the SaaS product). we do this because we don't want to expose certain things, like the `demo` org slug here

Reply:
you can see the component correlated with the `component:superuser-warning` hook here in getsentry